cmd/utils/cmd.go

```diff
@@ -63,14 +68,20 @@ func Fatalf(format string, args ...interface{}) {
 	os.Exit(1)
 }
 
-func StartNode(stack *node.Node) {
+func StartNode(ctx *cli.Context, stack *node.Node, chainID *big.Int) {
 	if err := stack.Start(); err != nil {
 		Fatalf("Error starting protocol stack: %v", err)
 	}
 	go func() {
 		sigc := make(chan os.Signal, 1)
 		signal.Notify(sigc, syscall.SIGINT, syscall.SIGTERM)
 		defer signal.Stop(sigc)
+
+		syncMode := *GlobalTextMarshaler(ctx, SyncModeFlag.Name).(*downloader.SyncMode)
+		if chainID.Int64() == 1 && syncMode != downloader.FastSync {
```

Comment:
It's the `light` sync mode we want to make an exception for. 

Still, I'm not 100% sure this is the best approach, I'll think about it.